src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -417,48 +418,48 @@ private static void validateAllCheckTokensAreReferencedInConfigFile(String confi
                 throw new CheckstyleException("Couldn't find check: " + checkName, ex);
             }
 
-            if (instance instanceof AbstractCheck) {
-                final AbstractCheck check = (AbstractCheck) instance;
-                if (isAllTokensAcceptable(check)) {
-                    // we can not have in our config test for all tokens
-                    continue;
-                }
-
-                Set<String> configTokens = configCheckTokens.get(checkName);
+            if (instance instanceof AbstractCheck
+                    && !isAllTokensAcceptable((AbstractCheck) instance)) {
+                // we can not have in our config test for all tokens
```

Comment:
This comment is for `continue` statement.

Reply:
Done